x-pack/legacy/plugins/apm/public/new-platform/plugin.tsx

```diff
@@ -29,7 +29,7 @@ const MainContainer = styled.div`
   min-height: calc(100vh - ${topNavHeight});
 `;
 
-function App() {
+const App = React.memo(() => {
```

Comment:
does `Rect.memo` really help in this case? I was understanding that it only re-renders the component if there is a change in props, but to my knowledge, the `App` component takes no props.

Reply:
It helps to not re-render App every time the location changes. This much I've verified. How much that actually helps in terms of performance impact, I'm not sure. AFAIK this happens because the upstream context providers re-render. See also https://twitter.com/sophiebits/status/1143199152030961665

Reply:
I'm removing this, it's either creating or surfacing some bug around redirects. I'll look into after merging though, if it's a bug that existed before, I'll submit a fix.